org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeThingHandler.java

```diff
@@ -368,6 +369,10 @@ private synchronized void doNodeInitialisation() {
                     continue;
                 }
 
+                if (channel.getConfiguration().get(ZclReportingConfig.CONFIG_POLLING) == null) {
```

Comment:
I am wondering why this is only done for the polling period here, but not for the other configuration parameters (reportingmax, reportingmin, reportingchange). Is this just because, currently, the reporting configuration is only used for the polling interval (and this needs, hence, to be extended as soon as some converter also makes use of the other reporting configuration options)?

Reply:
This is the feature we discussed elsewhere - if the reporting configuration fails to be set, then we have to update the polling period. Polling period gets set to different values following the initialisation - the reporting is not changed.

Reply:
OK, thanks for the info!